client/README.rst

```diff
@@ -1,5 +1,56 @@
 Quipucords UI
 =============
 
-User interface for Quipucords. This project was bootstrapped with [Create React App](https://github.com/facebookincubator/create-react-app).
+User interface for Quipucords. This project was bootstrapped with `Create React App <https://github.com/facebookincubator/create-react-app>`_.
+
+Requirements
+------------
+Before developing for Quipucords UI, some basic guidelines:
+ * Your system needs to be running `NodeJS version 6+ <https://nodejs.org/>`_
+ * To run the mock API, you'll need to be running `Docker <https://docs.docker.com/engine/installation/>`_
+
+Development
+-----------
+
+Installing
+^^^^^^^^^^
+1. Begin by cloning the repository::
+
+    $ git clone git@github.com:quipucords/quipucords.git
+
+2. Within the quipucords repo, install project dependencies::
+
+    $ cd client
+    $ npm install
+
+3. Setup and run the mock API::
+
+    $ npm run api:mock
+
+Development Server & Mock API
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+In order to run the development server use::
+
+    $ npm run start:client
+
+Unit Tests
+^^^^^^^^^^
+To run the unit tests use::
+
+    $ npm test
+
+Building the UI
+^^^^^^^^^^^^^^^
+The UI is compiled for production within a `client` directory underneath `./quipucords/client`::
+
+    $ npm run build:prod
+
+
+Serving the UI
+^^^^^^^^^^^^^^
+If you've gone through the Python quipucords installation you can serve the UI and API::
+
+    $ npm run start:quipucords
+
+*This NPM script is not intended as a replacement for serving the application, but a shortcut during development.*
```

Comment:
either (ultra breezy)
...but as a shortcut during development. 
or (still a breeze)
...but is intended to be used as a shortcut during development. 
or (more doc speak)
This NPM script is not intended to be used as a replacement for serving the application, but is intended to be used as a shortcut during development.